odb_extractor_atoms.py

```diff
@@ -286,7 +288,7 @@ def printseq(sequence, comment='', csv=False, path=''):
         if 'GMOS' in inst:
             coadds = '1'
             # convert wavelength to microns
-#             wavelength = '{:5.3f}'.format(float(wavelength) / 1000.)
+        #     wavelength = '{:5.3f}'.format(float(wavelength) / 1000.)
```

Comment:
I'm not quite clear why we're converting `wavelength` to  a float and then back to a `str`.
Do we need to do this for precision?

We should avoid using `.format` when possible as it is outdated and string interpolation is the preferred `f`-strings:
```
wavelength = f'{float(wavelength) /  1000:5.3f}'
```
I'm also unclear as to why we want the `5`, in the `:5.3f`, which doesn't seem to affect anything from what I can tell.

If we want to convert to microns, though, we could probably just do:
```
# Read wavelength in as nanometers and convert to micron
wavelength = f'{float(step['instrument:observingWavelength']) / 1000:.3f}'
```
I'm not even certain we require the `float()`... do we?

Reply:
Nix that last bit... it does look like we need the `float(...)` call as confirmed by the json file.